src/frontend/src/catalog/system_catalog/information_schema/schemata.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2024 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use risingwave_common::types::Fields;
+use risingwave_frontend_macro::system_catalog;
+
+/// The view schemata contains all schemas in the current database that the current user has access to (by way of being the owner or having some privilege).
```

Comment:
Should we filter the result by the current user in a more strict way? Not familiar with this system catalog,  since we don't support function like `has_schema_privilege` in rw, we can ignore that part for now.
It's also acceptable to me to keep the current impl.

Reply:
Cool, Let me add it